lib/manager/git-submodules/extract.ts

```diff
@@ -115,16 +115,16 @@ export default async function extractPackageFile(
           let httpSubModuleUrl = getHttpUrl(subModuleUrl);
           const hostRule = hostRules.find({ url: httpSubModuleUrl });
           httpSubModuleUrl = getHttpUrl(subModuleUrl, hostRule?.token);
-          const submoduleBranch = await getBranch(
+          const currentValue = await getBranch(
             gitModulesPath,
             name,
             httpSubModuleUrl
           );
           return {
             depName: path,
-            registryUrls: [httpSubModuleUrl, submoduleBranch],
+            lookupName: httpSubModuleUrl,
             currentValue,
-            currentDigest: currentValue,
+            currentDigest,
```

Comment:
My plan was to extract `branch` and `path` out to `managerData`. Similar to how it's done with other `git-tags` based datasources like [`Kustomize`](https://github.com/renovatebot/renovate/blob/master/lib/manager/kustomize/extract.ts#L21-L47) or [`ansible-galaxy`](https://github.com/renovatebot/renovate/blob/master/lib/manager/ansible-galaxy/extract.ts#L50-L58). Though, there are several managers that use `git-tags` datasource that have the exact same regex to extract the URL, so I can handle this in a refactoring after, if this PR unblocks other work like the URL refactoring.